blog/2022-09-30-mdx-blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: getting-started
+title: How to get started with sern!
+authors: [Sr Izan]
+tags: [guides]
+---
+
+## Hello everyone!
+
+I'm Sr Izan, your fellow sern-handler user and contributor.
+
+Today I'm going to show you how to get started with sern and all its cool features.
+
+### Step 1: Install the CLI.
+
+Just run:
+```
+npm i -g @sern/cli
+```
+then just use
```

Comment:
then run, not then just use. also add admonition for sern init -y

Reply:
if you want I can make it a tip instead

Reply:
done